src/main/java/bookstore/controller/BookController.java

```diff
@@ -28,24 +32,28 @@ public class BookController {
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(description = "Get all books")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
```

Comment:
You can add more information to your `@Operation` annotation. For example:
`@Operation(summary = "Get all books", description = "Get list of all books from DB")`

The same to othercases

Reply:
agree